src/lib/select/select.spec.ts

```diff
@@ -2917,11 +2917,15 @@ describe('MatSelect', () => {
       const topDifference = Math.floor(optionTop) -
           Math.floor(triggerTop - triggerFontSize - triggerExtraLineSpaceAbove);
 
+
+      console.log({optionTop});
+      console.log({triggerTop}, ' - ', (triggerTop - triggerFontSize - triggerExtraLineSpaceAbove));
+
       // Expect the coordinates to be within a pixel of each other. We can't rely on comparing
       // the exact value, because different browsers report the various sizes with slight (< 1px)
       // deviations.
-      expect(Math.abs(topDifference) < 2)
-          .toBe(true, `Expected trigger to align with option ${index}.`);
+      expect(Math.abs(topDifference))
+          .toBeLessThan(2, `Expected trigger to align with option ${index}.`);
```

Comment:
The old expectation was intentional. Changing it to `toBeLessThan` will log out something like `Expected 7 to be less than 2, Expected trigger to align with option 2.` which can be confusing.